src/test/fixtures/editMode.ts

```diff
@@ -248,8 +248,8 @@ export const awsCreateBlueprintRequest: CreateBlueprintRequest = {
 
 export const awsBlueprintResponse: BlueprintResponse = {
   ...awsCreateBlueprintRequest,
-  id: mockBlueprintIds['registration'],
-  description: mockBlueprintDescriptions['registration'],
+  id: mockBlueprintIds['aws'],
+  description: mockBlueprintDescriptions['aws'],
```

Comment:
Hehe this was the culprit :stuck_out_tongue:
We loaded wrong id on the blueprint get request and then we were assuming the aws blueptintID is different then what the unique request was returning and failing to allow saving the changes :)

Reply:
That was probably my doing :face_with_peeking_eye: :sweat_smile: Thanks for digging into it!! :bug: